packages/tasit-action/src/ContractEventsSubscription.js

```diff
@@ -1,15 +1,29 @@
 import Subscription from "./Subscription";
 
 class ContractEventsSubscription extends Subscription {
+  #ethersContract;
   #contract;
 
   // Note: We're considering listening multiple events at once
   //    adding eventName, listener params to constructor.
-  constructor(contract) {
-    super(contract);
+  constructor(ethersContract, contract) {
```

Comment:
Do we have it as a goal to make it not require both of these in the constructor at some point? Like maybe the contract knows about the ContractEventsSubscription instead of vice versa?

For whatever convenience it may offer right now, it's still hard to explain from an design perspective why a `ContractEventsSubscription` would need both

Reply:
Deprecated. Contract object it now subscribing to the ContractEventsSubscription error event.

Reply:
Great. Resolving.